.github/workflows/containerd.yml

```diff
@@ -71,6 +99,14 @@ jobs:
           make install
         working-directory: ${{ github.workspace }}/src/github.com/containerd/containerd
 
+      - name: Install ${{matrix.runc}} runtime engine on Linux
+        if: startsWith(matrix.os, 'ubuntu')
+        env:
+          RUNC_FLAVOR: ${{matrix.runc}}
+        run: |
+          gosu root script/setup/install-runc
```

Comment:
Is `gosu root` required here?  or can you just call script/setup/install-runc and it works successfully? I see containerd ci has it, just wondering if it is no longer required.

Reply:
Good point. Checked and not needed, so just calling the script direct.